source/dub/recipe/packagerecipe.d

```diff
@@ -247,8 +247,11 @@ struct BuildSettingsTemplate {
 					}
 
 					foreach (d; dirEntries(path.toNativeString(), pattern, SpanMode.depth)) {
-						import std.path : baseName;
-						if (baseName(d.name)[0] == '.' || d.isDir) continue;
+						import std.path : baseName, pathSplitter;
+						import std.algorithm.searching: canFind;
+						if (baseName(d.name)[0] == '.' || d.isDir ||
+							pathSplitter(d).canFind!(name => name[0] == '.'))
```

Comment:
Maybe sth. like:

```suggestion
							pathSplitter(d).canFind!(name => name.startsWith('.') && name != ".." && name.length >= 2)
```

Though it probably is better to make this a separate function, s.t. we can add more tester easier.